src/module.c

```diff
@@ -6538,7 +6550,7 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
             addACLLogEntry(ctx->client, acl_retval, ACL_LOG_CTX_MODULE, -1, c->user->name, object);
             if (error_as_call_replies) {
                 /* verbosity should be same as processCommand() in server.c */
-                sds acl_msg = getAclErrorMessage(acl_retval, c->user, c->cmd, c->argv[acl_errpos]->ptr, 0);
+                sds acl_msg = getAclErrorMessage(acl_retval, user, c->cmd, c->argv[acl_errpos]->ptr, 0);
```

Comment:
It goes to the inconsistency of using CtxUser and ClientUser.  Before there was only one (that got assigned to the client). Now, it can be one or the other, so abstracted out to "user".

so not a bug from before, but goes to the API issues discussed.